### DISCUSSION TITLE
increase UI text size on linux / use system DPI setting

### DISCUSSION THREAD

**Discussion by eadmaster** [dotnet/runtime#100711] [Q&A] [answered]
on my Kubuntu   22.04 machine i have set a global text scale of 125%, however this does not seem to affect dotnet6 apps:

![scale](https://github.com/dotnet/runtime/assets/925171/09b4255c-14de-4c49-8156-56c17499e008)

is there an envvar / global setting  for dotnet apps?

**Comment by danmoseley** [comment]
What UI tech is this - Maui, WPF, Winforms...? We can then move this discussion to the matching repo.

**Comment by eadmaster** [comment, marked as answer]
~~[WPF](https://github.com/SourMesen/Mesen2/blob/master/UI/app.manifest)~~ It's using Avalonia

EDIT: [seems related to this](https://github.com/AvaloniaUI/Avalonia/issues/9390)

fixed via [envvar](https://github.com/AvaloniaUI/Avalonia/wiki/Configuring-X11-per-monitor-DPI):
````
export AVALONIA_SCREEN_SCALE_FACTORS='HDMI-1=1.25'
````
